badgerdb/badgerdb.go

```diff
@@ -97,6 +98,38 @@ type Options struct {
 	// Directory for storing the DB files.
 	// Optional ("BadgerDB" by default).
 	Dir string
+	// Uses in-memory storage, ignores file-specified options.
+	// Optional (false by default).
+	InMemory bool
+	// When SyncWrites is true all writes are synced to disk.
+	// Optional (false by default).
+	SyncWrites bool
+	// When ReadOnly is true the DB will be opened on read-only mode.
+	// Multiple processes can open the same Badger DB.
+	// Optional (false by default).
+	ReadOnly bool
+	// Truncate indicates whether value log files should be truncated to delete corrupt data, if any.
```

Comment:
Is truncating really only for deleting _corrupt_ data? Haven't checked the BadgerDB docs yet, I guess the option is just passed and described in the BadgerDB docs the same way? Just asking to make sure :)

Reply:
https://github.com/dgraph-io/badger/blob/master/options.go#L273
Yes.